packages/core/src/types/redocly-yaml.ts

```diff
@@ -225,7 +225,10 @@ const ConfigRootTheme: NodeType = {
 const Rules: NodeType = {
   properties: {},
   additionalProperties: (value: unknown, key: string) => {
-    if (key.startsWith('assert/')) {
+    if (key.startsWith('rule/')) {
+      return 'Assert';
+    } else if (key.startsWith('assert/')) {
+      // keep the old assert/ prefix as an alias
```

Comment:
Don't we want to highlight that this is not expected in the config (both in the terminal and VS Code extension)?

Reply:
I don't know how to warn in this context. It is still valid, but not forever. It would be really good if it didn't autocomplete in the editor, but it isn't exactly an error either ...

Reply:
If you want to avoid erroring in VS Code, it's ok for now. Having the deprecation messages should be enough.